doc/docs/api/retool.md

```diff
@@ -0,0 +1,42 @@
+---
+sidebar_position: 99
+---
+
+# Retool
+
+
+`Retool` - is the backoffice tool. Ship contains predefined Retool apps. To find more about Retool please visit the [official site](https://retool.com/).
+
+
+## Installation guideline
+1. If you don't have Retool account yet, [sign up](https://login.retool.com/auth/signup).
+
+2. Import predefined Retool app. To import Retool app go to `Apps` > `Create New` > `From JSON`. Select the app from [retool directory](https://github.com/paralect/ship/tree/master/services/api-koa/src/config/retool). Give a name to your app.
+Look at [Predefined Apps](#predifined-apps) section to see available apps.
+
+![Import App](/img/api/retool/import-project.gif)
+
+3. Attach resources to the Retool app. To use Retool queries you need to attach resources to them. Select the query. Click on `Resource` select input. Select an existing resource or create a new one. Look at [App Resources](#app-resources) section to see how to connect the resources needed.
+![Update Resource](/img/api/retool/update-resources.gif)
+
+
+4. Enjoy the result. Read [Retool docs](https://docs.retool.com/docs/home) for more.
+
+
+## Predefined Apps
+### Users Management App
+Users Management App can be used for user administration. The main features of Users Management App are: 
```

Comment:
The application is built for user administration, it can be extended and used as internal admin panel. The main features are: